fastmri/data/mri_data.py

```diff
@@ -260,16 +265,24 @@ def __init__(
         else:
             dataset_cache = {}
 
+        # obtain annotation
+        if annotations is True:
+            annotation_fileName = self.Download_csv(annotations_version)
```

Comment:
We should first check if we've already downloaded the requested version. If we already downloaded, we don't need a new download.

Also, Python coding style does not use camel case. Variable names should be `annotation_file_name` and `download_csv`.

I see inside `download_csv` that you do have some checks for if it's already downloaded - in that case the function is not named properly.

Reply:
Yes, I will check it in download_csv().